mlrun/db/base.py

```diff
@@ -624,3 +624,11 @@ def set_run_notifications(
 
     def watch_log(self, uid, project="", watch=True, offset=0):
         pass
+
+    @abstractmethod
+    def get_datastore_profile(self, decorated_profile_name: str):
+        pass
+
```

Comment:
Looks like `delete_datastore_profile` and `list_datastore_profile` should be declared here as well but are missing.